imu_sensor_broadcaster/src/imu_sensor_broadcaster_parameters.yaml

```diff
@@ -41,3 +43,13 @@ imu_sensor_broadcaster:
       fixed_size<>: [9],
     }
   }
+  tf_frame_prefix_enable: {
+    type: bool,
+    default_value: true,
+    description:  "Enables or disables appending tf_prefix to tf frame id's.",
```

Comment:
I don't understand the need for this new parameter. Can you explain me why it is needed ?

Reply:
I followed the example of diff_drive. I think you can get rid of it.